lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -269,10 +281,23 @@ def main():
     cert_chain = module.params.get('cert_chain')
     dup_ok = module.params.get('dup_ok')
     if state == 'present':
-        cert = open(module.params.get('cert'), 'r').read().rstrip()
```

Comment:
Hi @s-hertel.

I'm happy to get this one merged so I can rebase and get some movement on https://github.com/ansible/ansible/pull/20622.  Ultimately this was @wrouesnel's PR though.

I note https://github.com/ansible/ansible/commit/62fa2019c6846def66912b94ba1b5ed0c40525d still has some of the issues rasied here.  [Line 270](https://github.com/ansible/ansible/blob/62fa2019c6846def66912b94ba1b5ed0c40525d0/lib/ansible/modules/cloud/amazon/iam_cert.py#L270) still defines `chain` which is never used and should be `cert_chain`. [Line 260](https://github.com/ansible/ansible/blob/62fa2019c6846def66912b94ba1b5ed0c40525d0/lib/ansible/modules/cloud/amazon/iam_cert.py#L260) could be removed if this was fixed.  I guess I'll fix those in https://github.com/ansible/ansible/pull/20622


Unfortunately there seems to be a bit of inconsistency in reviews.  On https://github.com/ansible/ansible/pull/20622 I was asked to remove empty `aliases: []` and `required: false` lines as part of my commit, yet there is no mention of it on this PR and on https://github.com/ansible/ansible/commit/62fa2019c6846def66912b94ba1b5ed0c40525d only the empty `aliases: []` entries have been removed.